slices/node-busboy.yaml

```diff
@@ -0,0 +1,7 @@
+package: node-busboy
+
+slices:
+  libs:
+    contents:
+      /usr/share/nodejs/busboy/lib/**:
```

Comment:
I think you're missing `/usr/share/nodejs/busboy/package.json`?

Reply:
> I think you're missing `/usr/share/nodejs/busboy/package.json`?

I have audited the code, and `busboy` does not need the `package.json` to function correctly.
Since the user application will not (and should not) use any Node modules from the system, `package.json` is not required.

Reply:
Ok, so what about all the other `package.json` files being included from different modules?

Reply:
> Ok, so what about all the other `package.json` files being included from different modules?

Some other packages require `package.json` to work (certain logic depends on definitions in `package.json`).
This practice is usually not recommended since it increases the bundle size during the deployment process.
However, many packages on NPM still do this.

Reply:
alright thanks for the explanation. Given the unpredictable rationale behind the inclusion of such files, I'll ask you to add comments on where those `package.json` files are "missing", such that we can remember the reasoning in the future if someone proposes adding those.

Reply:
I have now included all the `package.json` files for consistency.
Also, it might make more sense for unbuilt/non-bundled JavaScript modules to have a `package.json` file for the entry point information.